samples/SocialSample/Startup.cs

```diff
@@ -136,7 +136,7 @@ k web
                         JObject user = JObject.Parse(text);
 
                         var identity = new ClaimsIdentity(
-                            context.Options.AuthenticationType,
+                            context.Options.AuthenticationScheme,
```

Comment:
ClaimsIdentity has a constructor that takes just 'AuthencationType' no reason to pass defaults for Role and Name 
public ClaimsIdentity(string authenticationType)